modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -386,6 +386,11 @@ TEST_P(Test_ONNX_layers, ReduceL2)
     testONNXModels("reduceL2");
 }
 
+TEST_P(Test_ONNX_layers, Split)
+{
+    testONNXModels("split");
```

Comment:
There is no `split` tests in this PR. There are `split_1`, `split_2` and `split_3`. See https://github.com/opencv/opencv_extra/pull/701

```
[ RUN      ] Test_ONNX_layers.Split/0, where GetParam() = OCV/CPU
unknown file: Failure
C++ exception with description "OpenCV(3.4.9-dev) /build/precommit_linux64/3.4/opencv/modules/ts/src/ts.cpp:1052: error: (-2:Unspecified error) OpenCV tests: Can't find required data file: dnn/onnx/models/split.onnx in function 'findData'
" thrown in the test body.
[  FAILED  ] Test_ONNX_layers.Split/0, where GetParam() = OCV/CPU (0 ms)
https://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/24578/steps/test_dnn/logs/stdio
```